frontend/docker-bake.hcl

```
@@ -0,0 +1,30 @@
+variable "TAG" {
+    default = "develop"
+}
+
+variable "REPOSITORY" {
+    default = "ghcr.io"
+}
+
+variable "DEBUG" {
+    default = "0"
+}
+
+group "default" {
+    targets = [
+        "blockchain-telemetry-frontend",
+    ]
+}
+
+target "base" {
+    dockerfile = "Dockerfile"
+    args = {
+      DEBUG = "${DEBUG}"
+    }
+    platforms = ["linux/amd64"]
+}
+
+target "blockchain-telemetry-frontend" {
+    inherits = ["base"]
+    tags = ["${REPOSITORY}/thxnet/blockchain-telemetry-frontend:${TAG}"]
+}
```

Let's try to use this, since we do not have `target base` in the `Dockerfile`.

```suggestion
target "blockchain-telemetry-frontend" {
    dockerfile = "Dockerfile"
    args = {
      DEBUG = "${DEBUG}"
    }
    platforms = ["linux/amd64"]
    tags = ["${REPOSITORY}/thxnet/blockchain-telemetry-frontend:${TAG}"]
}
```